requirements.txt

```diff
@@ -1,2 +1,4 @@
+boto3>=1.26
+pymongo>=4.3
```

Comment:
```suggestion
```

Reply:
I don't understand the above, do these get installed in some other manner during build?